books/free-programming-books-subjects.md

```diff
@@ -259,6 +271,28 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Theory and Applications for Advanced Text Mining](http://www.intechopen.com/books/theory-and-applications-for-advanced-text-mining)
 
 
+### Designing
```

Comment:
the first three books cover UI design or Graphical Design; maybe the subject is really UI, not design. The 4th is very different - it's more like computer system architecture.